.tutone.yml

```diff
@@ -323,18 +313,94 @@ packages:
       - name: EntityTagValueWithMetadata
       - name: EntityTagWithMetadata
       - name: EntityType
+
+      # List of entities
+      - name: Entity
+        interface_methods:
```

Comment:
Where is this being used currently?

Reply:
Alerts: https://github.com/newrelic/newrelic-client-go/pull/590/files/ab2f926234458e74e8f366081573e2e1cbdb9158#diff-c8fb5f06f26ca2ea1554da789ba1914c18e334991add8cbd66482c4568907a17R96

Reply:
Okay, no worries.  I was going to complain, but this seems like a reasonable step.